builtin/audit/file/backend_test.go

```diff
@@ -0,0 +1,75 @@
+package file
+
+import (
+	"os"
+	"strconv"
+	"testing"
+
+	"github.com/hashicorp/vault/audit"
+	"github.com/hashicorp/vault/helper/salt"
+)
+
+func TestAuditFile_fileModeNew(t *testing.T) {
+	salter, _ := salt.NewSalt(nil, nil)
+
+	modeStr := "0777"
+	mode, err := strconv.ParseUint(modeStr, 8, 32)
+
+	file := "auditTest.txt"
```

Comment:
This test was meant to ensure that a new file could be created in Factory with the correct mode. Would using TempFile still satisfy that?